Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/packages.config

```diff
@@ -1,6 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.AI.DirectML" version="1.5.1" targetFramework="native" />
-  <package id="Microsoft.AI.MachineLearning" version="1.9.1" targetFramework="native" />
+  <package id="directxtk12_desktop_2019" version="2022.5.10.1" targetFramework="native" />
+  <package id="Microsoft.AI.DirectML" version="1.8.0" targetFramework="native" />
+  <package id="Microsoft.AI.MachineLearning" version="1.11.0" targetFramework="native" />
+  <package id="Microsoft.ML.OnnxRuntime.DirectML" version="1.11.0" targetFramework="native" />
   <package id="Microsoft.Windows.CppWinRT" version="2.0.210930.14" targetFramework="native" />

```

Comment:
Consider using the latest:

Microsoft.AI.DirectML.1.9.0
Microsoft.ML.OnnxRuntime.DirectML.1.12.1
...

Reply:
Done.